development/using-windows.md

```diff
@@ -27,37 +27,31 @@ Luckily, there is a perfectly working CouchDB installation for Windows:
 Then go to `http://localhost:5984/_utils/#/setup` in Windows and do the single node setup. Once done head back to linux and confirm it works:
 
 ```
-scdf@PC:/mnt/c/Users/stefa/Code/medic$ curl http://localhost:5984/
+yrimal@PC: curl http://localhost:5984/
 {"couchdb":"Welcome","version":"2.3.1","git_sha":"c298091a4","uuid":"5f60350abaaa11c0131a5630e83ae979","features":["pluggable-storage-engines","scheduler"],"vendor":{"name":"The Apache Software Foundation"}}
 ```
 
 ## Installing NPM
+Start your WSL instance (Ubuntu), not WSL as they take you to two different defult path. 
 
 The default `npm` in linux is really old and doesn't have `npm ci`, which we need.
 
 Instead use [nvm](https://github.com/nvm-sh/nvm) to install a later version.
 
-Once you've installed nvm: `nvm install 8` (or a later version).
+Once you've installed nvm: `nvm install 8` (or a later version). We used v11.3.0 for this purpose. 
 
 ## Checking out the code
 
-I used linux's git to check out the code, though presumably git in Windows works just as well.
+We used git that's preinstalled with Ubuntu to check out the code.
 
-You can access you c drive (and other Window's drives) from `/mnt`. To make life easier consider a symlink in your linux home directory to your code in Windows. In your linux home directory:
+You can checkout cht code inside wsl itself. We'll checkout inside /home/username/medic directory. 
 
 ```
-scdf@PC:~$ mkdir /mnt/c/Users/<username>/Code
-scdf@PC:~$ ln -s /mnt/c/Users/<username>/Code
-scdf@PC:~$ cd Code
-scdf@PC:~$ git clone git@github.com:medic/medic.git
-scdf@PC:~$ cd medic
+yrimal@PC:~$ mkdir /home/<<username>>/medic && cd /home/<<username>>/medic
```

Comment:
I don't think you need to make the folder before the `git clone` command. You can just run clone and it will make the folder.